src/bin/travis/resources/centaur.wdl

```diff
@@ -6,6 +6,26 @@ task centaur {
     File token
 
     command<<<
+        # start mysql server
+        mysqld &
+        
+        # give time to server to start
+        git clone https://github.com/vishnubob/wait-for-it.git
+        cd wait-for-it
+        ./wait-for-it.sh -t 10 localhost:3306
```

Comment:
Yes :) It was recommended by bernick when I did the docker compose with mysql script thing